Conceptpower+Spring/src/test/java/edu/asu/conceptpower/app/db/CustomMatchers/PredicateReviewRequestMatcher.java

```diff
@@ -0,0 +1,35 @@
+package edu.asu.conceptpower.app.db.CustomMatchers;
+
+import org.mockito.ArgumentMatcher;
+
+import com.db4o.query.Predicate;
+
+import edu.asu.conceptpower.core.ReviewRequest;
+
+/**
+ * This class is a custom argument matcher to compare Predicate<ReviewRequest> classes
+ * 
+ * @author keerthivasan
+ *
+ */
+public class PredicateReviewRequestMatcher implements ArgumentMatcher<Predicate<ReviewRequest>> {
+ 
+    private Predicate<ReviewRequest> left; 
```

Comment:
bad variable name, what does left mean in this context?

Reply:
I named the two objects to be compared as left and right. Left is the member variable and right will be used inside the matches() method.

Reply:
give them more telling names that left/right